src/containers/CommentsList.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import SingleTweetComment from "../components/SingleTweetComment";
+import { tweetCommentsData } from "../MOCKS/singleTweetPage";
+
+export const CommentsList: React.FC = () => {
+  return (
+    <div>
+      <SingleTweetComment {...tweetCommentsData} />
```

Comment:
tweetCimmentsData it's array of objects with comment data, so all what you need is to render each object by passing it to singletweetcomment in map cycle

Reply:
Example:

![image](https://user-images.githubusercontent.com/94183891/185785272-0d436f56-fff4-438c-a23a-d1e8190cc69f.png)